Gruntfile.js

```diff
@@ -1,21 +1,129 @@
 module.exports = function( grunt ) {
 
 grunt.loadNpmTasks( "grunt-contrib-jshint" );
+grunt.loadNpmTasks( "grunt-contrib-connect" );
+grunt.loadNpmTasks( "grunt-selenium-server" );
 grunt.loadNpmTasks( "grunt-html" );
 grunt.loadNpmTasks( "grunt-jscs" );
 
+var target,
```

Comment:
@gnarf yeah i'm sure i would have switched it eventually. Just as a note however most of this will be going away. I'm working to integrate much of this into perfjankie / browser-perf directly since they are really dependencies of those. Specifically the selenium stuff.